src/pypromice/process/join_l3.py

```diff
@@ -202,16 +220,23 @@ def join_l3(config_folder, site, folder_l3, folder_gcnet, outpath, variables, me
         
         filepath = os.path.join(folder_l3, stid, stid+'_hour.nc')
         isNead = False
-        if station_info["project"].lower() in ["historical gc-net", "glaciobasis"]:
+        if station_info["project"].lower() in ["historical gc-net"]:
             filepath = os.path.join(folder_gcnet, stid+'.csv')
             isNead = True
-        if not os.path.isfile(filepath):            
-            logger.info(stid+' is from an project '+folder_l3+' or '+folder_gcnet)
+        if not os.path.isfile(filepath):
+            logger.info(stid+' was listed as station but could not be found in '+folder_l3+' nor '+folder_gcnet)
```

Comment:
I anticipated that this would trigger an error and raise an exception. If the station list specifies data files, I would have expected those files to exist.

Reply:
My intention was that if a historical station is listed but cannot be found, then the latest data (which can most likely be found because it was produced by pypromice) should still be loaded and written in a l3/sites file. So that people fetching the latest data are not affected if there is a mess-up with the historical files' path.